okapi/core.php

```diff
@@ -28,7 +28,7 @@
 function get_admin_emails()
 {
     $emails = array();
-    if (class_exists("okapi\\Settings"))
+    if (class_exists(Settings::class))
```

Comment:
In other words, in your version, the admin will never receive an email if parsing of `Settings` fails. In the original implementation - he would.